apps/web/lib/settings/icon-popover.tsx

```diff
@@ -78,23 +77,23 @@ const IconPopover = ({
 										wrapperClassName="search-border bg-transparent dark:bg-[#E7E7EA]"
 										placeholder="Search Icon"
 										className="px-0 dark:bg-[#E7E7EA] rounded-none"
-										trailingNode={
-											<Tooltip
-												enabled={true}
-												placement="top"
-												label="Upload your own icon"
-												className="mr-5"
-											>
-												<div className="relative cursor-pointer">
-													<DocumentUploadIcon className="w-5 h-5" />
-													<input
-														type="file"
-														className="absolute inset-0 opacity-0"
-														accept="image/svg+xml"
-													/>
-												</div>
-											</Tooltip>
-										}
+										// trailingNode={
```

Comment:
@paradoxe35 why below code is commented out? For what it was used and why we don't need it anymore?

Reply:
No, we don't need it.
  It's an input file to upload a custom icon for a task status, since that it's a bad idea, so I commented it out

Reply:
hm, why it's a bad idea to upload a custom icon for a task status!?
CC: @KostyaEver